src/components/events/event_info/EventInfoForm.js

```diff
@@ -134,22 +137,37 @@ export class EventInfoForm extends Component {
     this.props.close();
   }
 
+  /**
+   * Toggles the priority of an event.
+   */
+  togglePriority() {
+    let priority = this.state.priority.value;
+    priority = (priority + 1) % 3;
+    this.onSave(this.state.priority.name, this.state.priority.type, priority);
+  }
+
   /**************************** Visual component ****************************/
 
   render() {
     let fields = (
       <Table className="table-hover">
         <tbody>
-          {Object.keys(this.state).map((field) => (
-            <tr key={field}>{this.getField(this.state[field])}</tr>
-          ))}
+          {Object.keys(this.state)
+            .filter((field) => field !== 'priority')
+            .map((field) => (
+              <tr key={field}>{this.getField(this.state[field])}</tr>
+            ))}
```

Comment:
Ha! Hacky. Kind of gross, but I approve.

Reply:
Haha yeah... Not thrilled...